EDEngineer/Resources/Data/localization.json

```diff
@@ -1952,144 +1952,6 @@
       "de": "Bestandteil in keinem bekannten Bauplan!",
       "fr": null,
       "ru": null
-    },

```

Comment:
I dont understand how i removed that xD And how i put it back xD Sry ! Trying ...